app/assets/javascripts/event-components/overview-fullpage.js

```diff
@@ -0,0 +1,15 @@
+new fullpage('#fullpage', {
+  // licenseKey: ...  
+
+  autoScrolling:true,
+  scrollHorizontally: true,
+  scrollOverflow: true,
+  navigation: true,
+  navigationPosition: 'left',
+  navigationBulletNames: [
+  	'2018', '2017', '2015', '2014', '2013', '2012', '2011', '2010', '2009', '2008'
+  ]
+});
+
+//methods
+window.fullpage_api.setAllowScrolling(true);
```

Comment:
Newline at the end of the document
```suggestion
window.fullpage_api.setAllowScrolling(true);

```